connectors/sources/github.py

```diff
@@ -1196,9 +1196,25 @@ async def _remote_validation(self):
         _, headers = await self.github_client.post(  # pyright: ignore
             query_data=query_data, need_headers=True
         )
-        if "repo" not in headers.get("X-OAuth-Scopes"):  # pyright: ignore
-            msg = "Configured token does not have required rights to fetch the content"
+
+        scopes = headers.get("X-OAuth-Scopes", "")
+        scopes = [scope.strip() for scope in scopes.split(",")]
+        required_scopes = ["repo", "user", "read:org"]
```

Comment:
Nit, why not define `required_scopes` as a set, as I see it's always used as `set(required_scopes)`. You can also consider to define `scopes` as a set as well.